pkg/features/kube_features.go

```diff
@@ -606,6 +606,13 @@ const (
 	// the pod is being deleted due to a disruption.
 	PodDisruptionConditions featuregate.Feature = "PodDisruptionConditions"
 
+	// owner: @danielvegamyhre
+	// kep: https://kep.k8s.io/4017
+	// beta: v1.28
+	//
+	// Set pod completion index as a pod label for Indexed Jobs and StatefulSets.
```

Comment:
```suggestion
	// Set pod completion index as a pod label for Indexed Jobs.
```

I imagine you are going to do SS in another PR but the comment is incorrect currently.

Reply:
Updated